test/unit/components/termsoverlay.test.js

```diff
@@ -7,7 +7,7 @@ var expect = chai.expect;
 var TermsOverlay = require('../../../app/components/termsoverlay');
 
 describe('TermsOverlay', function () {
-  
+
   describe('render', function() {
     it('should console.warn when trackMetric not set', function () {
```

Comment:
yeah - just updating so it covers all required props and will fail if new required props are added but bit tested for